### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by annacuomo**

### CODE REVIEW

File: scripts/rv_expression_association/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+* biallelic SNVs
+* not HomRef
+* freq<5%
+
+To run:
+```
+analysis-runner --dataset "tob-wgs" \
+    --description "count TOB rare variants" \
+    --access-level "standard" \
+    --output-dir "v0" \
```
Comment (lgruen) [inline scripts/rv_expression_association/README.md:12]: It doesn't matter much in this particular case, but please use something more descriptive than that for the top-level directory.

**Review by lgruen — APPROVED** [review]

File: scripts/rv_expression_association/README.md (hunk 1 above)
Comment (annacuomo) [reply scripts/rv_expression_association/README.md:12]: just in the command or the README more generally?

**Comment by MattWellie** [comment]
you should be able to fix that linting issue by adding 

```cpg_workflows>=1.1.4```

to requirements-dev.txt in the project root, and changing the vep/main.py

```from cpg_utils.workflows.batch import get_batch```

to

```from cpg_workflows.batch import get_batch```

**Review by annacuomo — COMMENTED** [review]
Added more detail to the README

**Pull request merged**